packages/victory-candlestick/src/index.js

```diff
@@ -1,2 +1,3 @@
 export { default as VictoryCandlestick } from "./victory-candlestick";
 export { default as Candle } from "./candle";
+export * as helperMethods from "./helper-methods";
```

Comment:
I don't think we need to include this export here - we should be able to change the import in the test from `victory-candlestick/es/helper-methods` to `victory-candlestick/lib/helper-methods`.